### ISSUE TITLE
Voidwalker pet behaves oddly against specific caster NPCs

### ISSUE THREAD

**Issue by CMHWoody** [chromiecraft/chromiecraft#7067]
The warlock's voidwalker pet will very quickly and continuously circle specific NPC caster types. This will make the voidwalker appear to just rotate insanely around the NPC.  It does not seem to happen with melee NPCs that I've noticed nor does it happen against all caster NPCs from what I've noticed.

It was most noticeable and consistently replicated when sending your pet against Darkeye Bonecaster in Tirisfal Glades. You will see the voidwalker do this rotation thing (very very quickly and constantly) around the NPC. 

Sometimes it does a slightly different, but still odd, jumping around thing that is not specifically rotating.

**Comment by CMHWoody** [comment]
Just had the same behavior against only the caster mobs in the Skull Rock just outside Org.  Voidwalker rotated around the caster enemies quickly.

**Comment by sogladev** [comment]
I've encountered the same issue with my voidwalker on some caster mobs on CC.

I managed to reproduce this reliably on acore 3d95ba220adf

1. send pet to attack
2. taunt to gain aggro, this causes the pet to reposition behind the mob
3. let pet gain aggro (taunt wears off in video)
4. pet moves inside the mob
5. observe spin

https://github.com/chromiecraft/chromiecraft/assets/46423958/c52c221e-0abb-46f8-be22-77a900360cbf


commands
```
heal
.cast self 1908
taunt
.learn 70428
soulshard
.add 6265 10
Fallen Ranger
.npc add 16314
voidwalker
.learn 697
felhunter
.learn 691
felguard
.learn 30146
```
